plugins/woocommerce/tests/php/includes/class-woocommerce-test.php

```diff
@@ -6,6 +6,24 @@
  * Unit tests for the WooCommerce class.
  */
 class WooCommerce_Test extends \WC_Unit_Test_Case {
+
+	/**
+	 * The default URI.
+	 *
+	 * @var string
+	 */
+	private static $default_uri;
```

Comment:
What about renaming this to `$origin_uri` instead? I think it makes it more clear? WDYT?

Reply:
Not a strong opinion, but I used this name mostly for consistency with another unit test:


https://github.com/woocommerce/woocommerce/blob/81414555912d0e8e6d10a7fbe5d3d823ec012647/plugins/woocommerce/tests/php/includes/admin/class-wc-admin-functions-test.php#L28-L45

I'm happy to change as well! Let me know what you think!

Reply:
Not a particularly strong opinion for me either just something I noted so not a blocker.